selenium-log4j/src/main/java/io.sterodium.extensions/logging/Logging.java

```diff
@@ -0,0 +1,133 @@
+package io.sterodium.extensions.logging;
+
+import org.apache.log4j.ConsoleAppender;
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.apache.log4j.PatternLayout;
+import org.apache.log4j.RollingFileAppender;
+import org.openqa.grid.common.CommandLineOptionHelper;
+import org.openqa.grid.common.exception.GridConfigurationException;
+import org.slf4j.LoggerFactory;
+import org.slf4j.bridge.SLF4JBridgeHandler;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collections;
+
+
+class Logging {
+
+    private static final org.slf4j.Logger LOG = LoggerFactory.getLogger(Logging.class);
+
+    private static final String LOG_FILE_PARAM = "-log";
+    private static final String LOG_FILE_PROPERTY = "selenium.LOGGER";
+
+    private static final String PATTERN = "%d{ISO8601} [%t] [%p] [%c] %m%n";
```

Comment:
Thread name is logged (appears to be unique). User object is not applicable here.